Frontend.Tests/PagesTests/Projects/Features/InitiatedTests.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Threading.Tasks;
+using Data;
+using Data.Models;
+using Data.Models.Projects;
+using Frontend.Models;
+using Frontend.Models.Features;
+using Frontend.Tests.Helpers;
+using Frontend.Validators.Features;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Routing;
+using Moq;
+using Xunit;
+
+namespace Frontend.Tests.PagesTests.Projects.Features
+{
+    public class InitiatedTests : PageTests
+    {
+        private readonly Pages.Projects.Features.Initiated _subject;
+
+        public InitiatedTests()
+        {
+            _subject = new Pages.Projects.Features.Initiated(ProjectRepository.Object)
+            {
+                Urn = ProjectUrn0001
+            };
+        }
+
+        [Fact]
+        public async void GivenUrn_GetsProjectFromRepositoryAndAssignsToTheView()
+        {
+            var request = new Func<Task<IActionResult>>(async () => await _subject.OnGetAsync());
```

Comment:
Is this different to just `await _subject.OnGetAsync();`?

Reply:
Good point, weird not sure why it was like that,